requirements.txt

```diff
@@ -2,4 +2,3 @@ numpy ~= 1.17
 click >= 6
 matplotlib ~= 3.1
 pytest ~= 6.0
-re ~= 2.0
```

Comment:
I would group this with the other removals from the requirements.txt and use a commit message like "remove modules that are not needed from requirements.txt"